app/builders/messages/messenger/message_builder.rb

```diff
@@ -2,7 +2,7 @@ class Messages::Messenger::MessageBuilder
   include ::FileTypeHelper
 
   def process_attachment(attachment)
-    return if attachment['type'].to_sym == :template
```

Comment:
Does this create empty messages? Should we do the filtering at the message creation step itself?

Reply:
Let me check with just attachments and no content.